modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -231,10 +237,24 @@ export default class Tile3DLayer extends CompositeLayer {
     } = tileHeader.content;
     const {positions, normals, colors} = attributes;
 
-    return (
-      positions &&
-      new PointCloudLayer({
-        id: `3d-point-cloud-tile-layer-${tileHeader.contentUri}`,
+    if (!positions) {
+      return null;
+    }
+
+    const {getColor} = this.props;
+    const SubLayerClass = this.getSubLayerClass(
+      `3d-point-cloud-tile-layer-${tileHeader.contentUri}`,
+      PointCloudLayer
+    );
+
+    return new SubLayerClass(
+      {
+        pointSize: 2.0
```

Comment:
We should allow users to customize this.

Reply:
I am thinking of providing a default value here. Also I thought `this.getSubLayerProps` picked up the props values from user?

Reply:
Yes this one definitely needs to be customizable. If the sublayer props system can handle it that will be great. If we explain carefully in the 3d-tile doc how sublayer props work we don't need another prop on the Tile3dLayer itself